ledger/sandbox/src/main/scala/com/digitalasset/platform/store/Conversions.scala

```diff
@@ -142,8 +142,9 @@ object Conversions {
 
   implicit def offsetToStatement: ToStatement[Offset] = new ToStatement[Offset] {
     override def set(s: PreparedStatement, index: Int, v: Offset): Unit =
-      s.setBinaryStream(index, v.toInputStream)
+      s.setBinaryStream(index, v.bytes.toInputStream)
```

Comment:
Same as previously, we can make use of the methods directly on `Offset`.

Reply:
fixed